javascript/ql/lib/semmle/javascript/frameworks/Handlebars.qll

```diff
@@ -27,3 +27,86 @@ module Handlebars {
     SafeString() { this = any(Handlebars h).getAConstructorInvocation("SafeString") }
   }
 }
+
+/** Provides logic for taint steps for the handlebars library. */
+module TaintStep {
+  /**
+   * Gets a `SourceNode` tracked from a compilation of a Handlebars template.
+   */
+  private DataFlow::SourceNode compiledHandlebarsTemplate(DataFlow::Node originalCall) {
+    result = compiledHandlebarsTemplate(DataFlow::TypeTracker::end(), originalCall)
+  }
+
+  private DataFlow::SourceNode compiledHandlebarsTemplate(
+    DataFlow::TypeTracker t, DataFlow::Node originalCall
```

Comment:
```suggestion
  private DataFlow::SourceNode compiledTemplate(DataFlow::CallNode compileCall) {
    result = compiledHandlebarsTemplate(DataFlow::TypeTracker::end(), compileCall)
  }

  private DataFlow::SourceNode compiledTemplate(
    DataFlow::TypeTracker t, DataFlow::CallNode compileCall
```
You might as well have a more specific type (`CallNode` instead of just `Node`).  
And you can rename `originalCall` to something more descriptive in the same go, I went for `compileCall`. 
We're in a `Handlebars` module in a `Handlebars` file, so I don't think we need to call this predicate something with `Handlebars`.

Reply:
👍